RoyalCar/Program.cs

```diff
@@ -15,36 +15,50 @@ public class Program
     {
         static void Main(string[] args)
         {
+            Dictionary<String, String> dictionary = new Dictionary<String, String>();
```

Comment:
the client application isn't a good place to put the dictionary. it's better to move it into a newly created project. also, seems like it should be stored in the file along with the cars. the name "dictionary" doesn't describe what it's used for, which is required for variable names